src/main/java/RankEmployeesUseCase/RankController.java

```diff
@@ -0,0 +1,23 @@
+package RankEmployeesUseCase;
+
+import Entity.CommonUser;
+import RankEmployeesUseCase.RankInputBoundary;
+import RankEmployeesUseCase.RankRequestModel;
+import RankEmployeesUseCase.RankResponseModel;
+
+import java.util.List;
+
+public class RankController {
+    private final RankInputBoundary userInput;
+
+    public RankController(RankInputBoundary userInput){
+        this.userInput = userInput;
+    }
+
+    RankResponseModel create(List<CommonUser> subordinates){
```

Comment:
The create method in Controller has an entity parameter. Although this does not violate the dependency inversion principle, the controller should not handle the entity directly. Could refactor the code later to use some attribute to replace the entity, and has a getter method in interactor to get the List of CommonUser by the new attribute.